osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -385,7 +382,7 @@ private Vector2 computePositionInStack(Vector2 position, float displayRadius)
         private void addLighting(JudgementResult judgementResult, Color4 colour, float x) =>
             hitExplosionContainer.Add(new HitExplosionEntry(Time.Current, judgementResult, colour, x));
 
-        private CaughtObject getCaughtObject(PalpableCatchHitObject source)
+        private CaughtObject? getCaughtObject(PalpableCatchHitObject source)
```

Comment:
Gonna note in passing that this one is a bit weird as I don't get how this can ever actually return null (although usage pattern makes *some* sense).